### DISCUSSION TITLE
ServoView | SharedArrayBuffer, WebGPU, WASM

### DISCUSSION THREAD

**Discussion by NYBACHOK** [servo/servo#34487] [Q&A] [answered]
Hi, I want to use servo for PoC Android application and I need 3 things(SharedArrayBuffer, WebGPU, WASM) to work. 
As I see, SharedArrayBuffer not implemented currently?

**Comment by jdm** [comment, marked as answer]
Correct. The JS engine implements SgaredArrayBuffer (https://doc.servo.org/mozjs_sys/jsapi/JS/struct.RealmCreationOptions.html) but Servo [disables that option](https://github.com/servo/servo/blob/c3a58f9c16895aaf137949944f4986ef67b2b68d/components/script/dom/bindings/interface.rs#L148) and does not implement any code that knows how to handle a SharedArrayBuffer from the JS engine.

**Reply by NYBACHOK** [reply]
Thanks for reply
